src/rest_api/api/search/substructure_search.rs

```diff
@@ -69,39 +69,41 @@ pub fn v1_index_search_substructure(
     if results.len() < limit {
         let tautomers = get_tautomers(&query_canon_taut);
 
-        let max_tauts = 10;
+        if tautomers.len() > 1 {
+            let max_tauts = 10;
```

Comment:
@Javier how do you feel about making max_tautomers an optional part of the REST API call? then Steven can tune it at will. and you can default it to 10? And if they set it to 0 there's no tautomer expansion?

Reply:
Alrighty, I've added this as an optional input argument. Also, I switched to using HashSets for the final results to ensure that using tautomers doesn't somehow end up duplicating results.